docs/getting-started/github-apps/github-apps.rst

```diff
@@ -66,6 +66,29 @@ to associate with a workflow on Dockstore. A template for both workflows and ser
 along with explanations for each field. For every branch/release on GitHub that has one of these files, a corresponding entry
 will be made on Dockstore.
 
+Error Handling
```

Comment:
I know you only moved this section, but I think this should really mention to look at the GitHub app logs as one of the first things to do.